universal-application-tool-0.0.1/app/models/Question.java

```diff
@@ -50,6 +50,16 @@ public Question(QuestionDefinition questionDefinition) {
     questionType = questionDefinition.getQuestionType().toString();
   }
 
+  public Question(long version, String path, String name, String description, ImmutableMap<Locale, String> questionText, ImmutableMap<Locale, String> questionHelpText, String questionType) {
```

Comment:
Suggestion: It's a lot of arguments to pass around individually. Another alternative is to create a play form object specifically for question creation and editing. ID would be optional on it since it won't exist when the form is submitted for create.

https://www.playframework.com/documentation/2.8.x/JavaForms#Defining-a-form